src/index.html

```diff
@@ -17,11 +17,365 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link references a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files, so you need to compile this SCSS file into a CSS file and link to the resulting CSS file instead.